### DISCUSSION TITLE
jsdelivr挂了

### DISCUSSION THREAD

**Discussion by L-20021213** [Qexo/Qexo#153] [Q&A] [answered]
我Fork了仓库，然后把仓库里的jsdelivr的链接全换成了反代的，但是重新构建之后还是https://cdn.jsdelivr.net/，不知道哪里的问题

**Comment by Fgaoxing** [comment]
![image](https://user-images.githubusercontent.com/92655031/169028635-02f10c9f-ea3a-43ad-9ca9-8bee33f5c2ed.png)

**Comment by am-abudu** [comment, marked as answer]
使用Dev分支并删除数据库重新部署

**Reply by am-abudu** [reply]
另：如需修改CDN请修改高级设置中的`CDN_PREV`项
